packages/react-native-session-replay/android/src/main/kotlin/com/datadog/reactnative/sessionreplay/ReactNativeSessionReplayExtensionSupport.kt

```diff
@@ -6,22 +6,55 @@
 
 package com.datadog.reactnative.sessionreplay
 
+import android.annotation.SuppressLint
+import android.util.Log
 import android.view.View
+import androidx.annotation.VisibleForTesting
 import com.datadog.android.sessionreplay.ExtensionSupport
 import com.datadog.android.sessionreplay.SessionReplayPrivacy
 import com.datadog.android.sessionreplay.internal.recorder.OptionSelectorDetector
 import com.datadog.android.sessionreplay.internal.recorder.mapper.WireframeMapper
+import com.datadog.reactnative.sessionreplay.mappers.ReactTextMapper
+import com.datadog.reactnative.sessionreplay.mappers.ReactViewGroupMapper
+import com.facebook.react.bridge.ReactContext
+import com.facebook.react.uimanager.UIManagerModule
+import com.facebook.react.views.text.ReactTextView
+import com.facebook.react.views.textinput.ReactEditText
 import com.facebook.react.views.view.ReactViewGroup
 
-internal class ReactNativeSessionReplayExtensionSupport : ExtensionSupport {
+internal class ReactNativeSessionReplayExtensionSupport(
+    private val reactContext: ReactContext,
+) : ExtensionSupport {
 
     override fun getCustomViewMappers(): Map<SessionReplayPrivacy, Map<Class<*>, WireframeMapper<View, *>>> {
-        return mapOf(SessionReplayPrivacy.ALLOW to mapOf(
-            ReactViewGroup::class.java to ReactViewGroupMapper() as WireframeMapper<View, *>
-        ))
+        val uiManagerModule = getUiManagerModule()
+
+        return mapOf(
+            SessionReplayPrivacy.ALLOW to mapOf(
+                ReactViewGroup::class.java to ReactViewGroupMapper() as WireframeMapper<View, *>,
+                ReactTextView::class.java to ReactTextMapper(reactContext, uiManagerModule) as WireframeMapper<View, *>,
+                ReactEditText::class.java to ReactTextMapper(reactContext, uiManagerModule) as WireframeMapper<View, *>
+            )
```

Comment:
very minor, to avoid repeating `WireframeMapper<View, *>`:

```suggestion
            SessionReplayPrivacy.ALLOW to mapOf(
                ReactViewGroup::class.java to ReactViewGroupMapper(),
                ReactTextView::class.java to ReactTextMapper(reactContext, uiManagerModule),
                ReactEditText::class.java to ReactTextMapper(reactContext, uiManagerModule)
            ).mapValues { it.value as WireframeMapper<View, *> }
```

or maybe specifying the signature in the `mapOf` will help to remove this case.